profile/README.md

```diff
@@ -65,20 +65,29 @@ The catchment area boundaries, uniquely generated by Crosscut, with all correspo
-Children under 5| The total number of children under 5 estimated to live within the given catchment area, based on [Meta's Data for Good](https://dataforgood.facebook.com/dfg/tools/high-resolution-population-density-maps) gridded population estimates 
-People 15 to 24| The total number of people aged 15 to 24 estimated to live within the given catchment area, based on [Meta's Data for Good](https://dataforgood.facebook.com/dfg/tools/high-resolution-population-density-maps) gridded population estimates 
-Women 15 to 49| The total number of women aged 15 to 49 estimated to live within the given catchment area, based on [Meta's Data for Good](https://dataforgood.facebook.com/dfg/tools/high-resolution-population-density-maps) gridded population estimates 
-People 60 and older| The total number of people aged 60 and over estimated to live within the given catchment area, based on [Meta's Data for Good](https://dataforgood.facebook.com/dfg/tools/high-resolution-population-density-maps) gridded population estimates 
-Buildings, OpenStreetMap| The total count of buildings estimated to be within the catchment area, based on [OSM Buildings](https://osmbuildings.org/)
-Google Buildings 60+% Confidence| The total count of buildings estimated to be within the catchment area, based on Google's [Open Buildings](https://sites.research.google/open-buildings/#faq), counting only those features rated with a 60% or greater confidence that they are buildings
-Google Buildings 70+% Confidence| The total count of buildings estimated to be within the catchment area, based on Google's [Open Buildings](https://sites.research.google/open-buildings/#faq), counting only those features rated with a 70% or greater confidence that they are buildings
-Google Buildings 80+% Confidence| The total count of buildings estimated to be within the catchment area, based on Google's [Open Buildings](https://sites.research.google/open-buildings/#faq), counting only those features rated with a 80% or greater confidence that they are buildings
-Average Building Footprint (m2) | Average square meters of the land area covered by each building in the catchment area based on Google's [Open Buildings](https://sites.research.google/open-buildings/#faq), counting only those features rated with a 80% or greater confidence that they are buildings
+| Name | The name associated with the catchment area |
+| Site Location | The latitude and longitude of the site associated with the catchment area (if applicable) |
+| Admin 0 | The admin 0 name associated with the catchment area |
+| Admin 1 | The admin 1 name(s) associated with the catchment area |
+| Admin 2 | The admin 2 name(s) associated with the catchment area (if applicable) |
+| Admin 3 | The admin 3 name(s) associated with the catchment area (if applicable) |
+| Admin 4 | The admin 4 name(s) associated with the catchment area (if applicable) |
```

Comment:
Would add: "Multiple names will be given if the catchment area crosses multiple administrative boundaries."